media/css/externalpages/pocket/components/_platform-footer.scss

```diff
@@ -0,0 +1,124 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.platform-footer {
+    border-top: 1px solid #cacaca;
```

Comment:
We should either create a variable holding this hexcode, or use a pre-defined one. `$color-marketing-gray-30` from Protocol looks very similar to it, so you can use that here

Reply:
Same thought with changing color to a protocol hex - I'm wary to change it. But I can bring it up with Craig/Alex

Reply:
Ok - just talked with Craig about it. Since these codes are apart of the pocket "theme" they should be hard coded until these hexes are added to protocol.